bika/lims/browser/templates/bika_listing_table_items.pt

```diff
@@ -421,7 +421,7 @@ Table cells for each column from in review_state's column list.
     <!-- Row Remarks field (for Analyses) -->
     <tal:remarks_condition
         tal:define="remarksenabled python:view.context.bika_setup.getEnableAnalysisRemarks();
-                    isanalysis python:'obj' in item and item['obj'].portal_type == 'Analysis';
+                    isanalysis python:'obj' in item and item['obj'].meta_type == 'Analysis';
```

Comment:
Not sure about using `meta_type` instead of `portal_type`.

In any case, it will not work if the analysis is a `DuplicateAnalysis`. Might be better to check against `AbstractRoutineAnalysis` (https://github.com/naralabs/bika.lims/blob/wip/bika/lims/content/abstractroutineanalysis.py#L105).

Since `AbstractRoutineAnalysis` implements IAnalysis, the best approach IMO would be something like:
```
isanalysis python:'obj' in item and IAnalysis.providedBy(item['obj'])
```

Note that because `ReferenceAnalysis` inherits from `AbstractAnalysis` (instead of `AbstractRoutineAnalysis`), so remarks for `ReferenceAnalysis` objects will never be displayed. @rockfruit I suggest `AbstractAnalysis` to implement `IAnalysis` (`AbstractRoutineAnalysis` implements `IRoutineAnalysis`), specially because there are some adapters (such as ResultsOutOfRange) for `IAnalysis` that must work for `ReferenceAnalysis` too.